File: Javascript/ejercicio22.js
//Algoritmo ContadorPara
let n;
let suma;
let i;

n = prompt("Ingrese un número entero: ");
n = parseInt(n);
if (isNaN(n)) {
    console.log("¡No ingresaste un número válido!");
    alert("¡No ingresaste un número válido!");
} else if (n < 0) {
    console.log("La suma no está definida para números negativos.");
    alert("La suma no está definida para números negativos.");
} else {
    suma = 0;  // Inicia el número entero
    for (let i = 1; i <= n; i++) {
        suma = suma + i  // Sumar  enteros
    }//FinPara
    console.log("El factorial de ", n, " es ", suma);
    alert("El factorial de" + n + "es ", +suma);
}//FinSi
//FinAlgoritmo

File: Phython/practica/ejercicio22.py
#Algoritmo ContadorPara

entrada = input("Ingrese un número entero: ")

try:

    n = int(entrada)
    print("¡No ingresaste un número válido!")

    if n < 0:
        print("La suma no está definida para números negativos.")

    else:
        suma = 0 # Inicia el número entero
        for i in range(1, n+1):
         suma = suma + i  #Sumar  
    #FinPara
        print(f"El factorial de {n} es {suma}")

except ValueError:
    print("¡No ingresaste un número válido!")

